src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -278,7 +278,13 @@ public Build getLastBuild() {
     public <T extends Parameter> T addParameter(Class<T> type) {
         ensureConfigPage();
 
-        check(find(by.xpath("//input[@name='parameterized']")));
```

Comment:
Possibly. I just wanted to disturb the old code branch as little as possible. Do you consider that cleanup a requirement? I would like to get this _tested_ change in as soon as possible so tests will pass again; any further edits mean a delay.